nym-vpn-api/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[profile.release]
+panic = "abort"
+opt-level = "s"
+overflow-checks = true
+
+[profile.dev]
+panic = "abort"
+
+[workspace]
+
+resolver = "2"
+members = [
+    "vpn-api-server",
+    "vpn-api-requests",
+    "vpn-api-lib-wasm"
+]
+
+[workspace.package]
+authors = ["Nym Technologies SA"]
+repository = "https://github.com/nymtech/nym"
+homepage = "https://nymtech.net"
+documentation = "https://nymtech.net"
+edition = "2021"
+license = "GPL-3.0"
+
+[workspace.dependencies]
+async-trait = "0.1.80"
```

Comment:
remove those in favour of the main workspace?

Reply:
Same as other comment, I agree we should switch to workspace deps asap after it's been embedded into it